### DISCUSSION TITLE
the changes are not applied. Help me!

### DISCUSSION THREAD

**Discussion by Kips37** [community/community#147214] [New to GitHub]
Good afternoon. Guys, tell me: the developer handed over the site and disappeared. I made changes to the site, namely, I changed the phone numbers and mail. It was preserved and nothing happened. Tell me what to do? I'm a beginner...

**Comment by Passion-Over-Pain** [comment]
Good Morning
What site would you be referring to ? A more specific breakdown of the problem might help others to help you

**Comment by aflacake** [comment]
Make sure they don't set the private mode repo, or a GitHub page error (hold on). Let them make the decision to merge your fork

**Comment by Kips37** [comment]
https://ivmegaprint.com// Привет. Исправления в код сайта я внес. Но они не отображаются(сайт для конечного пользователя не изменился). ----------------
                        Кому: community/community ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [community/community] the changes are not applied. Help me! (Discussion #147214);
                        16.01.2025, 17:13, "Nazwa Shabrina Zain" ***@***.***>:
                        
Make sure they don't set the private mode repo, or a GitHub page error (hold on). Let them make the decision to merge your fork

**Comment by Kips37** [comment]
https://ivmegaprint.com/   Привет) Внес измнения в код сайта(номер телефона и адрес). Сохранился, но эти изменения не отображаются на сайте...----------------
                        Кому: community/community ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [community/community] the changes are not applied. Help me! (Discussion #147214);
                        13.01.2025, 10:18, "Tinotenda Mhedziso" ***@***.***>:
                        
Good Morning
What site would you be referring to ? A more specific breakdown of the problem might help others to help you
